src/cljs/main/broadfcui/page/workspace/method_configs/method_config_editor.cljs

```diff
@@ -56,21 +56,11 @@
        :payload new-conf
        :headers utils/content-type=json
        :on-done (fn [{:keys [success? get-parsed-response xhr]}]
-                  (if-not success?
-                    (do (comps/push-error (str "Exception:\n" (.-statusText xhr)))
-                        (swap! state dissoc :blocker))
-                    (if (= name (config "name"))
-                      (swap! state assoc :loaded-config (get-parsed-response false) :blocker nil)
-                      (endpoints/call-ajax-orch ;; TODO - make unified call in orchestration
-                        {:endpoint (endpoints/rename-workspace-method-config workspace-id config)
-                         :payload (select-keys new-conf ["name" "namespace" "workspaceName"])
-                         :headers utils/content-type=json
-                         :on-done (fn [{:keys [success? xhr]}]
-                                    (swap! state dissoc :blocker)
-                                    (if success?
-                                      ((:on-rename props) name)
-                                      (comps/push-error
-                                       (str "Exception:\n" (.-statusText xhr)))))}))))})))
+                  (swap! state dissoc :blocker)
+                  (if success?
+                    (do ((:on-rename props) name)
```

Comment:
I wish Clojure had a better way to do this. It always looks weird to me. We could solve it ourselves:
```clojure
;; in utils.cljs
(defn invoke [f & args]
  (apply f args))

;; here
(do (utils/invoke (:on-rename props) name)
    ...)
```
Anyway, just ranting. Code looks good.